react/src/providers/ApolloAuth0Provider.tsx

```diff
@@ -81,14 +89,25 @@ function ApolloAuth0Provider({ children }: { children: ReactNode }) {
 
   const client = new ApolloClient({
     cache,
-    // HINT: Ideally, only set this temporary to true for local debugging
+    // TODO: Ideally, only set this temporary to true for local debugging
```

Comment:
Is it possible to do `connectToDevTools: process.env.NODE_ENV === "development",` ?

Reply:
fixed it